packages/functional-tests/src/tests/reparent-test.ts

```diff
@@ -18,17 +18,20 @@ export default class ReparentTest extends Test {
     }
 
     public async run(): Promise<boolean> {
-        this.sceneRoot = MRESDK.Actor.CreateEmpty(this.app.context).value;
-        const runningTestPromise = this.runTest();
-        const timeout = setTimeout(() => this.running = false, 60000);
-        await runningTestPromise;
-        clearTimeout(timeout);
-        destroyActors(this.sceneRoot);
-        return true;
+        try {
+            this.sceneRoot = MRESDK.Actor.CreateEmpty(this.app.context).value;
+            const runningTestPromise = this.runTest();
+            const timeout = setTimeout(() => this.running = false, 60000);
+            await runningTestPromise;
+            clearTimeout(timeout);
+            return true;
+        } finally {
+            destroyActors(this.sceneRoot);
+        }
     }
```

Comment:
managing actors shouldn't be needed anymore, as functional test menu wrapper cleans up all actors on exit. It's fine to do it manually, but I think we agreed unnecessary, right?

Reply:
(just noting here because you added extra code around the destroy call)

---
In reply to: [254025155](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/173#discussion_r254025155) [](ancestors = 254025155)

Reply:
Oh that's right. I'll take this out.

---
In reply to: [254025313](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/173#discussion_r254025313) [](ancestors = 254025313,254025155)